lib/SGN/Controller/AJAX/CrossWishlist.pm

```diff
@@ -201,6 +201,8 @@ sub create_cross_wishlist_submit_POST : Args(0) {
     my $selected_plot_ids = decode_json $c->req->param('selected_plot_ids');
     my $test_ona_form_name = $c->config->{odk_crossing_data_test_form_name};
     my $separate_crosswishlist_by_location = $c->config->{odk_crossing_data_separate_wishlist_by_location};
+    print STDERR "ONA FORM NAME =".Dumper($ona_form_name)."\n";
+    print STDERR "ONA FORM ID =".Dumper($ona_form_id)."\n";
```

Comment:
remove these if no longer used

Reply:
I intentionally left these 2 lines there. The system has many issues, I often have to debug.